lib/sql/views/followed_gamers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE VIEW warzone.followed_gamers AS
```

Comment:
When we make changes to the views in the DB, do we need to update them here manually? or are they automatically updated?